lib/src/wip/autocomplete/popup.dart

```diff
@@ -50,53 +55,47 @@ class _PopupState extends State<Popup> {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: EdgeInsets.only(
-        left: min(widget.column, widget.editingWindowSize.width - width),
-        top: widget.row,
-      ),
-      child: ConstrainedBox(
-        constraints: BoxConstraints(maxHeight: height, maxWidth: width),
+    return Positioned(
+      left: _horizontalOverflow
+          ? widget.flippedOffset.dx
+          : widget.normalOffset.dx,
+      top: _verticalOverflow ? widget.flippedOffset.dy : widget.normalOffset.dy,
+      child: Container(
+        alignment:
+            _verticalOverflow ? Alignment.bottomCenter : Alignment.topCenter,
+        constraints: const BoxConstraints(
+          maxHeight: _maxHeight,
+          maxWidth: _maxWidth,
+        ),
+        // DecoratedBox is rendered differently.
```

Comment:
That exactly is the problem with `DecoratedBox`? Can you describe in the comment?